pkg/operator/ceph/cluster/nodedaemon/pruner.go

```diff
@@ -82,7 +82,7 @@ func (r *ReconcileNode) createOrUpdateCephCron(cephCluster cephv1.CephCluster) (
 		k8sutil.AppAttr: prunerName,
 	}
 	cronJobLabels[k8sutil.ClusterAttr] = cephCluster.GetNamespace()
-
+	tolerations := cephCluster.Spec.Placement.All().Tolerations
```

Comment:
```suggestion
	tolerations := cephCluster.Spec.Placement.All().Tolerations
```

you don't need to do this, you can pass toleration as a argument here https://github.com/rook/rook/blob/1cbccf8fa7f5cdd6cd75340a406efdc5589d4f67/pkg/operator/ceph/cluster/nodedaemon/reconcile.go#L222 and then use that toleration below.

Reply:
done